vnext/Microsoft.ReactNative.Cxx.UnitTests/JSValueReaderTest.cpp

```diff
@@ -298,6 +298,7 @@ TEST_CLASS (JSValueReaderTest) {
       if (propertyName == L"StringValue1") {
         TestCheck(ReadValue<std::string>(reader) == "");
         TestCheck(ReadValue<std::wstring>(reader) == L"");
+        TestCheck(ReadValue<winrt::hstring>(reader) == L"");
```

Comment:
>ReadValue [](start = 18, length = 9)

Could you also add tests for the WriteValue? The goal is to have unit tests for all public methods.